lib/src/getContents/index.js

```diff
@@ -14,7 +15,9 @@ function getContents(opt) {
 
     // read and pass full contents
     if (opt.buffer !== false) {
-      return bufferFile(file, cb);
+      // set encoding type if user specifies it
+      var options = {'encoding': opt.encoding};
+      return bufferFile(file, options, cb);
```

Comment:
@limdauto passing the whole thing down is what is done for everything, makes it easier to add new option behavior later on down the road